.circleci/config.yml

```diff
@@ -214,13 +214,30 @@ jobs:
             -f integration-tests/docker-compose.yml \
             up --abort-on-container-exit --force-recreate
       - run:
-         name: Run additional integration tests
+         name: 'Run "204" integration tests'
          command: |
            docker-compose --version
            docker-compose \
             -f integration-tests/docker-compose.yml \
             -f integration-tests/docker-compose.204.yml \
             up --abort-on-container-exit --force-recreate
+      - run:
+         name: 'Run "init_error" integration tests'
+         command: |
+           docker-compose --version
+           set +e # We need this so that the run doesn't exit after docker-compose
+           docker-compose \
```

Comment:
Not needed now, but you can use `timeout --kill-after=DURATION --signal=SIGNAL ...` to automatically kill a command with SIGNAL after DURATION seconds.

Reply:
Okay. Here with `--abort-on-container-exit` docker compose will shut down all services as soon as the `contile` service exits or if that doesn't happen after the `client` exits because the test ran.